### ISSUE TITLE
Unit test breakage on Windows

### ISSUE THREAD

**Issue by fsat** [typesafehub/conductr-cli#119]
Tested on Python 3.5.1 on Windows 7.

```
IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m unittest

...

======================================================================
ERROR: test_wait_timeout (conductr_cli.test.test_bundle_installation.TestWaitForUninstallation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_bundle_installation.py", line 299, in test_wait_timeout
    self.assertRaises(WaitTimeoutError, bundle_installation.wait_for_uninstallation, bundle_id, args)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 727, in assertRaises
    return context.handle('assertRaises', args, kwargs)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 176, in handle
    callable_obj(*args, **kwargs)
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\bundle_installation.py", line 25, in wait_for_uninstallation
    return wait_for_condition(bundle_id, is_uninstalled, 'uninstalled', args)
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\bundle_installation.py", line 50, in wait_for_condition
    installed_bundles = count_installations(bundle_id, args)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 917, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 976, in _mock_call
    result = next(effect)
StopIteration

======================================================================
ERROR: test (conductr_cli.test.test_shazar.TestIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_shazar.py", line 58, in test
    'Created digested ZIP archive at {}/tmp[a-z0-9_]{{6,8}}-[a-f0-9]{{64}}\.zip'.format(self.tmpdir)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 1283, in assertRegex
    expected_regex = re.compile(expected_regex)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\re.py", line 224, in compile
    return _compile(pattern, flags)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\re.py", line 293, in _compile
    p = sre_compile.compile(pattern, flags)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\sre_compile.py", line 536, in compile
    p = sre_parse.parse(p, flags)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\sre_parse.py", line 829, in parse
    p = _parse_sub(source, pattern, 0)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\sre_parse.py", line 437, in _parse_sub
    itemsappend(_parse(source, state))
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\sre_parse.py", line 524, in _parse
    code = _escape(source, this, state)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\sre_parse.py", line 388, in _escape
    raise source.error("incomplete escape %s" % escape, len(escape))
sre_constants.error: incomplete escape \U at position 34

======================================================================
ERROR: test (conductr_cli.test.test_shazar.TestIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_shazar.py", line 63, in tearDown
    remove(self.tmpfile.name)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\IEUser\\AppData\\Local\\Temp\\tmpysoxn1t8'

======================================================================
FAIL: test_file (conductr_cli.resolvers.test.test_uri_resolver.TestGetUrl)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\resolvers\test\test_uri_resolver.py", line 165, in test_file
    'bundle-1.0-e78ed07d4a895e14595a21aef1bf616b1b0e4d886f3265bc7b152acf93d259b5.zip', filename)
AssertionError: 'bundle-1.0-e78ed07d4a895e14595a21aef1bf61[34 chars].zip' != 'C:\\Users\\IEUser\\Workspace\\conductr-cl[78 chars].zip'
- bundle-1.0-e78ed07d4a895e14595a21aef1bf616b1b0e4d886f3265bc7b152acf93d259b5.zip
+ C:\Users\IEUser\Workspace\conductr-cli\bundle-1.0-e78ed07d4a895e14595a21aef1bf616b1b0e4d886f3265bc7b152acf93d259b5.zip
? +++++++++++++++++++++++++++++++++++++++


======================================================================
FAIL: test_wait_timeout (conductr_cli.test.test_bundle_installation.TestWaitForInstallation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_bundle_installation.py", line 178, in test_wait_timeout
    self.assertRaises(WaitTimeoutError, bundle_installation.wait_for_installation, bundle_id, args)
AssertionError: WaitTimeoutError not raised by wait_for_installation

======================================================================
FAIL: test_wait_timeout (conductr_cli.test.test_bundle_scale.TestWaitForScale)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_bundle_scale.py", line 207, in test_wait_timeout
    self.assertRaises(WaitTimeoutError, bundle_scale.wait_for_scale, bundle_id, 3, args)
AssertionError: WaitTimeoutError not raised by wait_for_scale

======================================================================
FAIL: test_failure (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 132, in test_failure
    self.base_test_failure()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 223, in base_test_failure
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_failure_install_timeout (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 292, in test_failure_install_timeout
    self.base_test_failure_install_timeout()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 392, in base_test_failure_install_timeout
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_failure_invalid_address (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 135, in test_failure_invalid_address
    self.base_test_failure_invalid_address()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 246, in base_test_failure_invalid_address
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_failure_no_response (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 138, in test_failure_no_response
    self.base_test_failure_no_response()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 268, in base_test_failure_no_response
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 70, in test_success
    self.base_test_success()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 72, in base_test_success
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_custom_ip_port (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 82, in test_success_custom_ip_port
    self.base_test_success_custom_ip_port()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 177, in base_test_success_custom_ip_port
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_long_ids (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 79, in test_success_long_ids
    self.base_test_success_long_ids()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 150, in base_test_success_long_ids
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_no_wait (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 129, in test_success_no_wait
    self.base_test_success_no_wait()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 203, in base_test_success_no_wait
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_quiet (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 76, in test_success_quiet
    self.base_test_success_quiet()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 124, in base_test_success_quiet
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_verbose (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 73, in test_success_verbose
    self.base_test_success_verbose()
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\conduct_load_test_base.py", line 98, in base_test_success_verbose
    http_method.assert_called_with(self.default_url, files=self.default_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'bundle.zip'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1))], timeout=30)

======================================================================
FAIL: test_success_with_configuration (conductr_cli.test.test_conduct_load_v1.TestConductLoadCommand)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\IEUser\Workspace\conductr-cli\conductr_cli\test\test_conduct_load_v1.py", line 122, in test_success_with_configuration
    http_method.assert_called_with(self.default_url, files=expected_files, timeout=LOAD_HTTP_TIMEOUT)
  File "C:\Users\IEUser\AppData\Local\Programs\Python\Python35-32\lib\unittest\mock.py", line 792, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: Expected call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0'), ('memory', '200'), ('diskSpace', '100'), ('roles', 'web-server'), ('bundleName', 'overlaid-name'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1)), ('configuration', ('config.zip', 1))], timeout=30)
Actual call: mock('http://127.0.0.1:9005/bundles', files=[('nrOfCpus', '1.0 '), ('memory', '200 '), ('diskSpace', '100 '), ('roles', 'web-server'), ('bundleName', 'overlaid-name'), ('system', 'bundle'), ('bundle', ('bundle.zip', 1)), ('configuration', ('config.zip', 1))], timeout=30)

----------------------------------------------------------------------
Ran 229 tests in 0.766s

FAILED (failures=14, errors=3)

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$
```

**Comment by fsat** [comment]
Also, the conduct commands work fine on Windows:

```
IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct stop --ip 192.168.99.100 vis
Bundle stop request sent.
Bundle 6e4c9f9fae2ab8f2f8e91a0e24ba9dfa waiting to reach expected scale 0
Bundle 6e4c9f9fae2ab8f2f8e91a0e24ba9dfa expected scale 0 is met
Unload bundle with: conduct unload --ip 192.168.99.100 6e4c9f9
Print ConductR info with: conduct info --ip 192.168.99.100

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct info --ip 192.168.99.100
ID       NAME                    #REP  #STR  #RUN
0e0602e  conductr-elasticsearch     1     0     1
c68bf1b  conductr-kibana            1     0     1
6e4c9f9  visualizer                 1     0     0

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct run --ip 192.168.99.100 vis
Bundle run request sent.
Bundle 6e4c9f9fae2ab8f2f8e91a0e24ba9dfa waiting to reach expected scale 1
Bundle 6e4c9f9fae2ab8f2f8e91a0e24ba9dfa has scale 0, expected 1
Bundle 6e4c9f9fae2ab8f2f8e91a0e24ba9dfa expected scale 1 is met
Stop bundle with: conduct stop --ip 192.168.99.100 6e4c9f9
Print ConductR info with: conduct info --ip 192.168.99.100

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct load --ip 192.168.99.100 reactive-maps-frontend
Retrieving bundle...
Loading bundle from cache typesafe/bundle/reactive-maps-frontend
Retrieving file:///C:\Users\IEUser\Workspace\conductr-cli\reactive-maps-frontend
Resolving bundle typesafe/bundle/reactive-maps-frontend
Retrieving https://dl.bintray.com/typesafe/bundle/typesafe/reactive-maps-frontend/v1-a5eb1554d40d92d9cf3c11ba121f5624f6bc4c73b482dabb6ef6a545168bbf/reactive-maps-frontend-v1-a5eb1554d40d92d9cf3c11ba121f5624f6bc4c73b482dabb6ef6a545da168bbf.zip
[#################################################] 100%
Loading bundle to ConductR...
Bundle a5eb1554d40d92d9cf3c11ba121f5624 waiting to be installed
Bundle a5eb1554d40d92d9cf3c11ba121f5624 installed
Bundle loaded.
Start bundle with: conduct run --ip 192.168.99.100 a5eb155
Unload bundle with: conduct unload --ip 192.168.99.100 a5eb155
Print ConductR info with: conduct info --ip 192.168.99.100

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct info --ip 192.168.99.100
ID       NAME                    #REP  #STR  #RUN
0e0602e  conductr-elasticsearch     1     0     1
c68bf1b  conductr-kibana            1     0     1
6e4c9f9  visualizer                 1     0     1
a5eb155  reactive-maps-frontend     1     0     0

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ conduct unload --ip 192.168.99.100 reactive-maps-frontend
Bundle unload request sent.
Bundle a5eb1554d40d92d9cf3c11ba121f5624 waiting to be uninstalled
Bundle a5eb1554d40d92d9cf3c11ba121f5624 uninstalled
Print ConductR info with: conduct info --ip 192.168.99.100

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct info --ip 192.168.99.100
ID       NAME                    #REP  #STR  #RUN
0e0602e  conductr-elasticsearch     1     0     1
c68bf1b  conductr-kibana            1     0     1
6e4c9f9  visualizer                 1     0     1

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct logs --ip 192.168.99.100 -n 100 vis
TIME      HOST          LOG
18:13:08  15ea3faf366c  [info] application - Signalled start to ConductR
18:13:08  15ea3faf366c  [WARN] [03/16/2016 01:13:08.799] [main] [ActorConfiguration$] Configuration does not define any actors to monitor
18:13:09  15ea3faf366c  [info] play.api.Play - Application started (Prod)
18:13:09  15ea3faf366c  [info] p.c.s.NettyServer - Listening for HTTP on /172.17.0.2:10609

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct events --ip 192.168.99.100 -n 100 vis
TIME      EVENT                                         DESC
18:05:09  conductr.loadExecutor.bundleWritten           Bundle written: requestId=b25c2055-fda6-4a0f-a56f-f8ba4f5cab67, bundleName=visualizer, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:05:34  conductr.loadScheduler.unloadBundleRequested  Unload bundle requested: requestId=e88ee97a-339a-4575-a851-8af275912b30, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:05:34  conductr.loadExecutor.bundleRemoved           Bundle removed: bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, uniqueAddress=UniqueAddress(akka.tcp://conductr@172.17.0.2:9004,-116150138)
18:05:50  conductr.loadExecutor.bundleWritten           Bundle written: requestId=fcd68605-b2ec-49ef-a009-cabc356d0178, bundleName=visualizer, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:06:16  conductr.loadScheduler.unloadBundleRequested  Unload bundle requested: requestId=7d007da7-0015-46e4-a7ff-ceed816e4d01, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:06:16  conductr.loadExecutor.bundleRemoved           Bundle removed: bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, uniqueAddress=UniqueAddress(akka.tcp://conductr@172.17.0.2:9004,-116150138)
18:11:41  conductr.loadExecutor.bundleWritten           Bundle written: requestId=ac6496af-b315-4893-a6ca-8433fd7416d3, bundleName=visualizer, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:13:06  conductr.scaleScheduler.scaleBundleRequested  Scale bundle requested: requestId=8ba6b2a6-c7fe-4372-8006-9724b63f45ff, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, scale=1
18:13:06  conductr.scaleScheduler.scaleBundleRequested  Scale bundle requested: requestId=daa35255-8b03-4d11-bd07-2e0fdd48ebee, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, scale=1
18:13:06  conductr.scaleScheduler.scaleRescheduled      Scale rescheduled: requestId=daa35255-8b03-4d11-bd07-2e0fdd48ebee, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa
18:13:07  conductr.launcher.bundleStarted               Bundle started: bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, host=172.17.0.2
18:13:08  conductr.scaleScheduler.scaleBundleRequested  Scale bundle requested: requestId=25edc6f4-d1a1-4df9-87e1-7daf2de1cd7c, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa, scale=1
18:13:08  conductr.scaleScheduler.scaleRescheduled      Scale rescheduled: requestId=25edc6f4-d1a1-4df9-87e1-7daf2de1cd7c, bundleId=6e4c9f9fae2ab8f2f8e91a0e24ba9dfa

IEUser@IE9Win7 MINGW32 ~/Workspace/conductr-cli (master)
$ python -m conductr_cli.conduct services --ip 192.168.99.100
SERVICE                      BUNDLE ID  BUNDLE NAME             STATUS
http://127.0.0.1:9200        0e0602e    conductr-elasticsearch  Running
http://:5601                 c68bf1b    conductr-kibana         Running
http://:9200/elastic-search  0e0602e    conductr-elasticsearch  Running
http://:9999                 6e4c9f9    visualizer              Running
tcp://:9300                  0e0602e    conductr-elasticsearch  Running
```

**Comment by fsat** [comment]
This is fixed as part of #120
